<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterCpWindow</class>
 <widget class="QMainWindow" name="MasterCpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Master CP</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mastercpclient.qrc">
    <normaloff>:/ControlPanel/Resources/quasilogo.ico</normaloff>:/ControlPanel/Resources/quasilogo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="startOrderButton">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mastercpclient.qrc">
           <normaloff>:/ControlPanel/Resources/play.png</normaloff>:/ControlPanel/Resources/play.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="stopOrderButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mastercpclient.qrc">
           <normaloff>:/ControlPanel/Resources/stop.png</normaloff>:/ControlPanel/Resources/stop.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="removeOrderButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mastercpclient.qrc">
           <normaloff>:/ControlPanel/Resources/remove.png</normaloff>:/ControlPanel/Resources/remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="addOrderButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mastercpclient.qrc">
           <normaloff>:/ControlPanel/Resources/add.png</normaloff>:/ControlPanel/Resources/add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="orderTable">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="HelpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="AboutAction"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionDebug"/>
   </widget>
   <addaction name="HelpMenu"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="AboutAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionReferals">
   <property name="text">
    <string>Referrals</string>
   </property>
  </action>
  <action name="actionQueue">
   <property name="text">
    <string>Queue</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mastercpclient.qrc"/>
 </resources>
 <connections/>
</ui>
